corehq/apps/app_manager/static/app_manager/js/multimedia_size_util.js

```diff
@@ -0,0 +1,44 @@
+hqDefine('app_manager/js/multimedia_size_util',[
+    'jquery',
+    'underscore',
+    'knockout',
+    'hqwebapp/js/initial_page_data',
+], function ($, _, ko, initialPageData) {
+    var multimediaSize = function (name, size) {
+        var self = {};
+        self.name = ko.observable(name);
+        self.size = ko.observable(size);
+        return self;
+    }
+    var multimediaSizesView = function (url) {
+        var self = {};
+        self.sizes = ko.observableArray();
+        self.load_state = ko.observable(null);
+        self.showSpinner = ko.observable(false);
+        self.load = function () {
+            self.load_state('loading');
+            $.ajax({
+                url: url,
+                success: function (content) {
+                    _.each(content, function(mmsize, mmType) {
+                        self.sizes.push(multimediaSize(mmType, mmsize));
+                    });
+                    self.load_state('loaded');
+                },
+                error: function (data) {
+                    if (data.hasOwnProperty('responseJSON')) {
+                        alert(data.responseJSON.message);
```

Comment:
Browser alerts are jarring. Can you use `hqwebapp/js/alert_user.js instead? It mimicks django's `messages`.

Reply:
https://github.com/dimagi/commcare-hq/pull/25864/commits/87f99658f2a4f3746254131c0e84ae9ad4ec4c87